packages/theme/src/components/container.ts

```diff
@@ -0,0 +1,11 @@
+const baseStyle = {
+    w: "100%",
+    mx: "auto",
+    maxW:"60ch",
+    px: "1rem",
+}
+
+export default {
+    baseStyle,
+}
+  
```

Comment:
Could you run prettier on this file again? I think there is too much whitespace in there.

Reply:
Done, turns out it was a windows vs linux (wsl) thing....